common/src/app/common/types/page/flow.cljc

```diff
@@ -0,0 +1,24 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) UXBOX Labs SL
```

Comment:
Should be `KALEIDOS INC`

Reply:
and the rest of files, ensure correct copyright holder

Reply:
Ok, but this should be in a different branch, because there are a lot of files with UXBOX Labs, unrelated tho this PR.

Reply:
yep, this is fixed on one of my pending to merge PR, but this is just for not introduce again incorrect string

Reply:
ah wel... Done, then.